### DISCUSSION TITLE
Gradle examples

### DISCUSSION THREAD

**Discussion by ahndmal** [hantsy/spring-reactive-sample#151] [Ideas]
Hello @hantsy !

Would it be possible to add Gradle-based files to the Spring examples ?

Thanks :)

**Comment by hantsy** [comment]
There are a lot of examples on github using Gradle, I also created many examples using Gradle, see https://github.com/hantsy/spring-graphql-sample/, etc. 
Gradle is just a build tool, it is easy to switch to Gradle from Maven.  

This repository will use Maven as build tools. There is no plan to add Gradle buildscripts.

**Reply by ahndmal** [reply]
@hantsy ,

Yes, sure. You are right. Just wanted to confirm about such an option.

Thanks for clarifying and for your work :)
